packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5574,13 +5583,11 @@ The follow provisioning actions are supported by each IdP provider:
 
 | Type         | User Provisioning Actions     | Group Provisioning Actions            |
 | ------------ | ----------------------------- | ------------------------------------- |
-| `FACEBOOK`   | `AUTO`, `CALLOUT`, `DISABLED` | `NONE` or `ASSIGN`                    |
-| `GOOGLE`     | `AUTO`, `CALLOUT`, `DISABLED` | `NONE` or `ASSIGN`                    |
-| `LINKEDIN`   | `AUTO`, `CALLOUT`, `DISABLED` | `NONE` or `ASSIGN`                    |
+| `GOOGLE`     | `AUTO`, `DISABLED` | `NONE` or `ASSIGN`                    |
 | `SAML2`      | `AUTO` or `DISABLED`          | `NONE`, `ASSIGN`, `APPEND`, or `SYNC` |
 | `X509`       | `DISABLED`                    | No support for JIT provisioning       |
 
-> **Note:** `CALLOUT` is a <ApiLifecycle access="deprecated" /> User provisioning action.
+> **Note:** All social IdPs (such as `FACEBOOK`, `LINKEDIN`, and so on) support the same User and Group Provisioning Actions as the `GOOGLE` IdP type.
```

Comment:
Why not put the same type as you did on line 5535?

```
[[all social IdP types *]](#social-idp-type-policy-actions)
```

instead of

```
GOOGLE
```

Reply:
This is just a different presentation for you guys to comment on ;-)
I'll change it to line 5535 format if that is preferred.